CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ v2.4.1 is a maintenance release with the following fixes and enhancements:
 
  - Removed usage of `strcpy` to enhance security of the client (#1745)
  - Fixed invalid write in `OAUTHBEARER/OIDC` extensions copy (#1745)
+ - [KIP-107](https://cwiki.apache.org/confluence/display/KAFKA/KIP-107%3A+Add+deleteRecordsBefore%28%29+API+in+AdminClient) Added delete_records API (#1710)
```

Comment:
1. Move this to top of the list.
2. Change version to v2.5.0 as we are adding a new feature. 
3. Change to `v2.4.1 is a maintenance release with the following fixes and enhancements:` to `v2.5.0 is a feature release with the following features, fixes and enhancements:`